src/node.h

```diff
@@ -677,7 +677,8 @@ NODE_EXTERN Environment* CreateEnvironment(
 NODE_EXTERN std::unique_ptr<InspectorParentHandle> GetInspectorParentHandle(
     Environment* parent_env,
     ThreadId child_thread_id,
-    const char* child_url);
+    const char* child_url,
+    const char* title_prefix);
```

Comment:
Labeled this as `semver-major` because adding a parameter here is breaking the ABI (and, since there is no default value for the parameter, also the API); you can avoid that by adding an overload with this signature instead of changing the signature.

(For example: c566a04c867544ee2d32c8ddfdf68122bd7ec498 introduced a new parameter to a C++ API function in an ABI-compatible way, e8bddac3e94866f2ca2ff5d064cd96c72be2d03c followed up as a semver-major change to simplify the implementation again)

Reply:
Ah understood! I think overloading would be better, updating!

Reply:
Updated, could you check